lib/scrabble.rb

```diff
@@ -1,7 +1,23 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word
+
+  def initialize(word)
+    @word = word
+  end
+
+  def score
+    @point_values(letter)
+        letters = letter.chars.map.upcase do |char|
+          point_values(letters)
+
+          total = 0
```

Comment:
If you set total to be 0 in this block, every time it loops through it will re-set the total to 0 instead of counting up.